javascript/ql/src/Performance/PolynomialReDoS.qhelp

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+
+<qhelp>
+
+	<include src="ReDoSIntroduction.qhelp" />
+
+	<example>
+		<p>
+
+			Consider this use of a regular expression, which removes
+			all leading and trailing whitespace a string:
```

Comment:
Missing word here:
Consider this use of a regular expression, which removes all leading and trailing whitespace in or inside a string?